lib/timescaledb/model/peer_metrics.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Model.PeerMetrics do
+  @moduledoc """
+  Model representing set of peer metrics from `Membrane.RTC.Engine` metrics report.
+  """
+
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          time: NaiveDateTime.t() | nil,
+          peer_id: String.t() | nil,
+          "peer.metadata": String.t() | nil,
+          "ice.binding_requests_received": non_neg_integer() | nil,
+          "ice.binding_responses_sent": non_neg_integer() | nil,
+          "ice.bytes_received": non_neg_integer() | nil,
+          "ice.bytes_sent": non_neg_integer() | nil,
+          "ice.packets_received": non_neg_integer() | nil,
+          "ice.packets_sent": non_neg_integer() | nil
+        }
+
+  @primary_key false
+  schema "peers_metrics" do
+    field(:time, :naive_datetime_usec)
+    field(:peer_id, :string)
```

Comment:
fields are usually defined without parenthesis. You should add `import_deps: [:ecto]` to `.formatter.exs` to prevent formatted from adding them